src/types/types.js

```diff
@@ -210,28 +210,20 @@ export type SpendingLimits = {
 }
 
 export type BuySellPlugin = {
+  id: string,
   pluginId: string,
+  priority: number,
+  paymentType: string,
+  description: string,
+  title: string,
+  partnerIconPath: string,
+  cryptoCodes: Array<string>,
   uri: string,
   name: string,
-  subtitle: string,
-  imageUrl: string,
   supportEmail?: string,
   permissions?: Array<string>,
   originWhitelist?: Array<string>,
-  isLegacy?: true
-}
-
-export type BuySellFilter = {
```

Comment:
Bring back this flow type and edit it to match the JSON objects for buy and sell plug-ins